packages/run-it/src/components/ShowResponse/responseUtils.tsx

```diff
@@ -26,6 +26,7 @@
 import React, { ReactElement } from 'react'
 import { IRawResponse, ResponseMode, responseMode } from '@looker/sdk-rtl'
 import { Paragraph, CodeBlock } from '@looker/components'
+import { CodeDisplay } from '@looker/code-display'
 
 import { CodeStructure } from '../CodeStructure'
```

Comment:
It looks like Prism supports html syntax highlighting so you can replace `CodeStructure` on L117 as well and get rid of the import. In the end we should be left with `CodeStructure` only in `RequestForm`